docs/UserGuide.md

```diff
@@ -38,14 +38,15 @@ Expense Expert is a **desktop app for keeping track of your expenses, and managi
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `ExpenseExpert.jar` from [here](https://github.com/AY2122S2-CS2103T-W09-3/tp/releases).
+2. Download the latest `ExpenseExpert.jar` from [here](https://github.com/AY2122S2-CS2103T-W09-3/tp/releases).
```

Comment:
Take note of naming, so if yall were to submit jar file, make sure the Es are capital.

Reply:
changed to small e's in the ug

Reply:
wait, if you changed to small e, means its not standardise? Here is big e.